lib/components/output-area.js

```diff
@@ -26,14 +26,25 @@ class OutputArea extends React.Component<{ store: store }> {
     this.showHistory.set(false);
   };
 
+  getOutputText(output: Object): ?string {
+    switch (output.output_type) {
+      case "stream":
+        return output.text;
+      case "execute_result":
+        return output.data["text/plain"];
+      case "error":
+        return output.traceback.toJS().join("\n");
+    }
+  }
+
   handleClick = () => {
     const kernel = this.props.store.kernel;
     if (!kernel || !kernel.outputStore) return;
     const output = kernel.outputStore.outputs[kernel.outputStore.index];
-    // check for a text property and fall back to data["text/plain"]
-    const textOrBundle = output.text || output.data["text/plain"];
-    if (textOrBundle) {
-      atom.clipboard.write(Anser.ansiToText(textOrBundle));
+    const copyOutput = this.getOutputText(output);
+
+    if (copyOutput) {
+      atom.clipboard.write(Anser.ansiToText(copyOutput));
```

Comment:
> Wouldn't you want to have anser still strip any ansi colored text or codes out?

I intended to do this here, but come to think of it this only needs to be done for the error case so it fits better up in the `getOutputText` function

Reply:
Oh I see, thanks.